distro/data/src/main/resources/data/apiman-realm-for-keycloak.json

```diff
@@ -108,17 +108,19 @@
             "enabled" : true,
             "baseUrl" : "/apimanui",
             "redirectUris" : [
-                "https://localhost:8443/apimanui/*"
+                "https://localhost:8443/apimanui/*",
+                "http://localhost:8080/apimanui/*"
             ],
             "secret" : "password",
-            "webOrigins" : [ "*" ]
+            "webOrigins" : [ "+" ]
```

Comment:
Is this a reference to this? https://lists.jboss.org/pipermail/keycloak-user/2017-October/012008.html

Reply:
Could be! heh
I made this change for security purposes, if we use '*' any weborigin is allowed to consume resources as a client, when I set to the '+' only valid redirects (set on the client configuration) is allowed as weborigins and it prevents unauthorized access and solves some CORS problem too, but CORS problem is not the case :).